Add schedule creation

jonaskaz opened the pull request:
Add a module to create schedules from a garden config

jonaskaz committed:
Add schedule creation

Add a module to create schedules from a garden config

tzou2024 commented on pkg/schedule.go:
```diff
@@ -0,0 +1,78 @@
+	Datetime int64   `json:"datetime"`
+}
+
+func strToTime(dtStr string) (time.Time, error) {
```
Little confused about how times are coming in and converted, so we are going to have timestamps in the OnTimes[] and those are used to generate new timestamps using the hour and minute etc? How does the logic know to generate a schedule for a specific day?

jonaskaz replied:
That's a good question, I wasn't really thinking about this. We should probably remove date from this and just use time instead. I also wasn't totally sure of the format for timestamps that will be returned by hydrangea, I'll need to double check that

jonaskaz committed:
Change datetime format to string time

This will make parsing downstream in gardener more straightforward
and avoid confusion with datetimes vs times in our scheduling
